spec/api-web-contents-spec.js

```diff
@@ -324,6 +324,21 @@ describe('webContents module', function () {
     })
   })
 
+  describe('getOSProcessId()', function () {
+    it('returns a valid procress id', function () {
+      // load URL otherwise getOSProcessId() returns 0
+      w.loadURL('file://' + path.join(__dirname, 'fixtures', 'pages', 'focus-web-contents.html'))
```

Comment:
You could just load `about:blank` here if you want to instead of this page, makes it a more concise and makes this spec independent of this fixture.